apps/nestjs-app/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "nestjs-app",
+  "version": "0.0.1",
+  "description": "",
+  "author": "",
+  "private": true,
+  "license": "UNLICENSED",
+  "dependencies": {
+    "@nestjs/common": "^9.0.0",
+    "@nestjs/core": "^9.0.0",
+    "@nestjs/platform-express": "^9.0.0",
+    "reflect-metadata": "^0.1.13",
+    "rxjs": "^7.2.0"
+  },
+  "devDependencies": {
+    "@nestjs/cli": "^9.0.0",
+    "@nestjs/schematics": "^9.0.0",
+    "@nestjs/testing": "^9.0.0",
+    "@types/express": "^4.17.13",
+    "@types/supertest": "^2.0.11",
+    "@typescript-eslint/eslint-plugin": "^5.0.0",
```

Comment:
remove these 4 eslint deps, theyll be inherited from root config

Reply:
I need `@nestjs/testing` and `supertest\ for `e2e` tests in the file `apps/nestjs-app/test/app.e2e-spec.ts`.
Should I create an 'e2e' task in the `moon.yml` of the project to run the following command: `jest --config ./test/jest-e2e.json`? This is the command that Nest provides when creating a new project in the scripts section of the `package.json`.

Reply:
Yeah that makes sense.